src/main/java/com/google/sps/models/TopChartsCluster.java

```diff
@@ -4,14 +4,15 @@
 import java.util.List;
 
 public class TopChartsCluster extends Cluster {
-    private String title = "Top Charts";
-    private String subtitle = "For Wear OS";
+    private String title;
+    private String subtitle;
     private List<Chart> charts = new ArrayList<Chart>();
 
-    TopChartsCluster() {
-        makeCluster();
+    public TopChartsCluster() {
         this.type = getType();
         this.id = getID();
+        this.title = "Top Charts";
```

Comment:
Is this not the constructor? I don't understand what you're asking.

Reply:
Like this.

```
public TopChartsCluster(String title) {
this.title = title
...
```

Reply:
So we need to pass in the title everytime we make a TopCharts object?

Reply:
Yes, since the title should be stored in the database. Not all Top Charts clusters are guaranteed to have the same title. I.e., they can be "Top Daily Charts", "Top Monthly Charts", "Top Kids Apps", "Top Movies", etc.

Reply:
I created a setTitle function because i felt that would be a better way to set the title from the db Is that ok?